nox/tasks.py

```diff
@@ -84,12 +86,17 @@ def filter_manifest(manifest, global_config):
             the manifest otherwise (to be sent to the next task).
 
     """
+
+    nox_env = os.environ.get("NOXSESSION")
```

Comment:
It is possible for us to do this in `__main__` using argparse?

https://github.com/theacodes/nox/blob/master/nox/__main__.py#L66

Reply:
I wasn't sure where to put it. A default arg should be enough I think